### DISCUSSION TITLE
Dispatching no longer provided?

### DISCUSSION THREAD

**Discussion by kaihendry** [BottlecapDave/HomeAssistant-OctopusEnergy#1097] [Q&A]
<img src=https://s.natalian.org/2024-11-25/dispatching.png>

My devices dependent on the cheap tariff are not working reliably: https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/discussions/1090

**Comment by BottlecapDave** [comment]
Hello. This usually happens if
A) the integration has not loaded correctly and would include errors in your HA logs
B) your account no longer states you're on an intelligent tariff
C) you've updated from a very old version where the name of the entity changed.

My guess would be option A, so please review the logs and post them here. A reload of the integration might also be required (the OE APIs need to be available on older versions of the integration on startup. Newer versions can read data from the cache)
